dev/shopify_product_taxonomy.gemspec

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name          = "shopify_product_taxonomy"
+
+  require_relative "lib/product_taxonomy/version"
+  spec.version       = ProductTaxonomy::VERSION
+  
+  spec.authors       = ["Shopify"]
+  spec.email         = ["gems@shopify.com"]
+
+  spec.summary       = "Load the complete Shopify Standard Product Taxonomy into memory as a tree "
+  spec.description   = "A code-only gem providing Ruby classes and utilities to parse and process " \
+                       "Shopify's Standard Product Taxonomy data " \
+                       "from YAML source files into an in-memory Ruby object model."
+  spec.homepage      = "https://github.com/Shopify/product-taxonomy"
+  spec.license       = "MIT"
+  spec.required_ruby_version = ">= 3.0"
+
+  spec.metadata = {
+    "bug_tracker_uri" => "https://github.com/Shopify/product-taxonomy/issues",
+    "source_code_uri" => "https://github.com/Shopify/product-taxonomy",
+    "allowed_push_host" => "https://rubygems.org",
+  }
+
+  spec.files         = Dir.glob("lib/**/*").reject { |f| f.start_with?("lib/product_taxonomy/commands") || f == "lib/product_taxonomy/cli.rb" }
```

Comment:
Don't you need to include the data files in this gem? Otherwise they will not be able to be loaded.

Reply:
The data files are excluded because:

- they're released on a different (more frequent) cadence from the loader code (gem)
- they would make the gem significantly bigger

Reply:
Just updated the `spec.description` to specify where the data files are kept.